Source/Task/AsyncLib.cpp

```diff
@@ -32,8 +31,17 @@ struct AsyncState
     const void* identity = nullptr;
     const char* identityName = nullptr;
 
-    AsyncState() noexcept :
-        timer{ this, TimerCallback }
+    void* operator new(size_t size, size_t additional)
+    {
+        return malloc(size + additional);
+    }
+
+    void operator delete(void* ptr)
+    {
+        free(ptr);
+    }
```

Comment:
Oh I see, couldn't you forward `operator new(size_t, size_t)` to `operator new(size_t)`? Just in case someone else overrides the global `operator new`

Reply:
Fixed.